### ISSUE TITLE
HHVM combatibility

### ISSUE THREAD

**Issue by hason** [bobthecow/psysh#81]
```
Psy Shell v0.1.11 (PHP 5.5.99-hiphop — cli) by Justin Hileman

Fatal error: Call to undefined method Closure$Psy\ExecutionLoop\Loop::run;1719123410::bindTo from context Psy\ExecutionLoop\Loop in /usr/local/share/php/vendor/psy/psysh/src/Psy/ExecutionLoop/Loop.php on line 109
```

Depends on facebook/hhvm#1203

**Comment by bobthecow** [comment]
As of HHVM 3.5.0 the loop binds properly, so that's been updated. I added a fix for another HHVM bug that kept builds from passing (see 7fd995d). But right now my dev machine doesn't have a working HHVM instance, and I spent an hour last night building and re-building and finally gave up. So HHVM tests still don't pass, is what I'm saying. I'd welcome pull requests from anyone who's into that sort of thing, though, because I don't think it'll take much to get things working.
